src/main/kotlin/org/acejump/session/Session.kt

```diff
@@ -231,13 +232,15 @@ class Session(private val editor: Editor) {
    * Should only be used from [SessionManager] to dispose a
    * successfully ended session.
    */
-  internal fun dispose() {
+  internal fun dispose(jumpPerformed: Boolean) {
```

Comment:
We could change this signature to `dispose(jump: TaggingResult.Jump?)`, remove `getMarkByOffset` and pass `calculateMarkAndQuery(jump)`.